org.octave.Octave.json

```diff
@@ -0,0 +1,377 @@
+          "url": "https://support.hdfgroup.org/ftp/HDF5/current18/src/hdf5-1.8.19.tar.bz2",
+          "sha256": "59c03816105d57990329537ad1049ba22c2b8afe1890085f0c022b75f1727238"
+        }
+      ]
+    },
+    {
+      "name": "qrupdate",
+      "make-install-args": [
+        "PREFIX=/app",
+        "BLAS=-lopenblas",
+        "LAPACK="
+      ],
+      "no-autogen": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/qrupdate/files/qrupdate/1.2/qrupdate-1.1.2.tar.gz",
+          "sha256": "e2a1c711dc8ebc418e21195833814cb2f84b878b90a2774365f0166402308e08"
+        }
+      ]
+    },
+    {
+      "name": "suitesparse",
+      "build-options": {
+        "cflags": "-L/app/lib"
+      },
+      "make-args": [
+        "LAPACK=",
+        "library"
+      ],
+      "make-install-args": [
+        "LAPACK=",
+        "INSTALL_LIB=/app/lib",
+        "INSTALL_INCLUDE=/app/include",
+        "library"
+      ],
+      "no-autogen": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.5.tar.gz",
+          "sha256": "b9a98de0ddafe7659adffad8a58ca3911c1afa8b509355e7aa58b02feb35d9b6"
+        }
+      ]
+    },
+    {
+      "name": "glpk",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnu.org/gnu/glpk/glpk-4.63.tar.gz",
+          "sha256": "914d27f1a51c2bf4a51f1bd4a507f875fcca99db7b219380b836a25b29b3e7f6"
+        }
+      ]
+    },
+    {
+      "name": "portaudio",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz",
+          "sha256": "f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513"
+        }
+      ]
+    },
+    {
+      "name": "graphicsmagick",
+      "config-opts": [
+        "--enable-shared=yes"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.26/GraphicsMagick-1.3.26.tar.bz2",
+          "sha256": "054bd6e10c6d82b6adb36d1a9f860e9db90689d3b61e77a344d2f962b33167e5"
+        }
+      ]
+    },
+    {
+      "name": "arpack-ng",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/opencollab/arpack-ng/archive/3.5.0.tar.gz",
+          "sha256": "50f7a3e3aec2e08e732a487919262238f8504c3ef927246ec3495617dde81239"
+        }
+      ]
+    },
+    {
+      "name": "fftw3",
+      "no-autogen": true,
+      "config-opts": [
+        "--enable-shared",
+        "--enable-threads"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.fftw.org/fftw-3.3.6-pl2.tar.gz",
+          "sha256": "a5de35c5c824a78a058ca54278c706cdf3d4abba1c56b63531c2cb05f5d57da2"
+        }
+      ]
+    },
+    {
+      "name": "fftw3f",
```

Comment:
Does it need to be build separately?

Reply:
Looks like it: http://www.fftw.org/fftw2_doc/fftw_6.html#SEC69

Reply:
They can be built together, but need two separate configure/make/make install incantations. For example a single build could run two different configures in separate build subdirs, if flatpak supports that kind of thing.

Reply:
I don't think so. :+1: then.